Unmappable character encoding in EvEfiBootServicesApp

On the CI build several  "unmappable character" errors are occuring:
```
HIRS/HIRS_Utils/src/main/java/hirs/tpm/eventlog/events/EvEfiBootServicesApp.java:30: error: unmappable character for encoding ASCII

 *    Type 0x01 ??? Hardware Device Path
```

--- a/HIRS_Utils/src/main/java/hirs/tpm/eventlog/events/EvEfiBootServicesApp.java
+++ b/HIRS_Utils/src/main/java/hirs/tpm/eventlog/events/EvEfiBootServicesApp.java
@@ -27,12 +27,12 @@ import hirs.utils.HexUtils;
  *    } EFI_DEVICE_PATH_PROTOCOL;    // ref page of the UEFI spec
  *
  *    Where Type and Subtype are defined the UEFI spec section 10.3.1
- *    Type 0x01 – Hardware Device Path
- *    Type 0x02 – ACPI Device Path
- *    Type 0x03 – Messaging Device Path
- *    Type 0x04 – Media Device Path
- *    Type 0x05 – BIOS Boot Specification Device Path
- *    Type 0x7F – End of Hardware Device Path
+ *    Type 0x01  Hardware Device Path
+ *    Type 0x02  ACPI Device Path
+ *    Type 0x03  Messaging Device Path
+ *    Type 0x04  Media Device Path
+ *    Type 0x05  BIOS Boot Specification Device Path
+ *    Type 0x7F  End of Hardware Device Path
  */
 public class EvEfiBootServicesApp {
    /** UEFI Address. */

